app/controllers/concerns/mfa_expiry_methods.rb

```diff
@@ -12,7 +12,9 @@ def delete_mfa_expiry_session
 
     def session_active?
       return false if session[:mfa_expires_at].nil?
-      session[:mfa_expires_at] > Time.current
+      return true if session[:mfa_expires_at] > Time.current
+      delete_mfa_expiry_session
+      false
```

Comment:
This saves us from some ugliness when the before_filter prevents rendering on an expired session, preventing the after action form running that would clear `mfa_expires_at`.

Reply:
please add a comment to that effect